srcpkgs/amule/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'amule'
+pkgname=amule
+version=11066
+revision=1
+wrksrc=amule-master
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man
+	--enable-cas --enable-wxcas --enable-amule-daemon
+	--enable-amulecmd --enable-amule-gui --enable-alc
+	--enable-alcc --enable-webserver --disable-debug
+	--enable-optimize --enable-geoip --disable-upnp
+	--enable-fileview --with-boost  --with-wx-config=/usr/bin/wx-config-gtk3"
+hostmakedepends="tar autoconf automake gettext gettext-devel wxWidgets-gtk3"
```

Comment:
Does it actually need tar (bsdtar is used by default to extract distfiles)?

Reply:
The `./autogen.sh` script called in the `pre_configure` hook invokes `tar`. If `tar` is not included as a host dependency, the hook fails on account of `tar` unavailable`.